docs/reference/expectations/result_format.md

```diff
@@ -32,7 +70,7 @@ cases for working with Great Expectations, including interactive exploratory wor
 |    missing_count                      |no              |yes             |yes             |yes             |
 |    missing_percent                    |no              |yes             |yes             |yes             |
 |    details (dictionary)               |Defined on a per-expectation basis                                 |
-| Fields defined for `column_map_expectation` type expectations:                                            |
+### Fields defined for `column_map_expectation` type expectations:                                            
```

Comment:
This won't render the table correctly.  By changing this to a header, you're breaking the table that was set up above without adding a header row to the new table section below.  You need to add the following below both of the places where you updated this to be a header:
```
| Fields within `result`                |BOOLEAN_ONLY    |BASIC           |SUMMARY         |COMPLETE        |
----------------------------------------|----------------|----------------|----------------|-----------------
```
That should make the sections you partitioned out render as tables.

Reply:
thanks @Rachel-Reverie!